### PR TITLE
Documentation Overhaul - September 2019

### PR DESCRIPTION
**Pull request by bmehendale**
Cleaning up documentation in general. I'll do this in several parts. This is Part 1.

### CODE REVIEW

File: docs/home/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 
-Esper Dev provides tools that enable Android developers to deliver highly secure solutions for Dedicated Devices with greater flexibility and control. This leads to a better development experience, designed for the entire development lifecycle, while helping IT do more with less. A [rich API](https://api.esper.io) set and a variety of [open-source SDKs](https://github.com/esper-io/esper-client-py) and tools allow you to incorporate device management for your apps and Dedicated Device apps and dedicated devices.\
+Esper provides tools that enable Android developers to deliver highly secure solutions to Dedicated Devices, all the while giving them superior flexibility and control over the process. This leads to a better development experience spanning the entire development lifecycle. Furthermore, this lightens the burder on IT, enabling them to focus on more pressing aspects of your product. A [rich API](https://api.esper.io) set and a variety of [open-source SDKs](https://github.com/esper-io/esper-client-py) and tools allow you to incorporate device management for your dedicated device apps and dedicated devices.
```
Comment (kirananto) [inline docs/home/README.md]: (typo) burder

File: docs/home/requirements.md [hunk 2]
```diff
@@ -1,55 +1,65 @@
 
 # Requirements
+
 <div class="avg-reading-time">Average reading time : {{ $page.readingTime.text.replace(' read', '') }}s</div>
 
-In order to use Esper Dev, the following are the minimum requirements covering the various tools available from Esper Dev.
+Following sections talk about the minimum requirements for using various parts of Esper.
+
+## Web-Based Console (*.esper.cloud)
```
Comment (kirananto) [inline docs/home/requirements.md]: You can use {{ esper_tenant_name }} here to have dynamic endpoint name

File: docs/home/requirements.md [hunk 3]
```diff
@@ -1,55 +1,65 @@
...
-## Device Requirements
-Targeted Dedicated Devices must run Android version 4.4 KitKat (API Level 19) or above. Alternatively, Esper Development Devices running Esper Enhanced Android can be used; this opens up additional features when using Esper Dev.
+Targeted Dedicated Devices must run Android version 4.4 KitKat (API Level 19) or above. However, we recommend that devices running Android version 7.0 Nougat (API Level 24) or above be used with Esper.
 
-Please make sure to whitelist the following URLs and Ports if your devices are behind any firewall with restrictions in placed for inbound and outbound traffic.
+Esper Development Devices running Esper Enhanced Android will always be supported be used. Such devices open up additional features for developers, when using Esper.
+
+## Infrastructure
+
+Dedicated devices are often behind corporate firewalls, preventing Esper backends to communite with them (or vice-versa). Please make sure to whitelist the following URLs and Ports, in case your fleet is running behind such a firewall.
```
Comment (kirananto) [inline docs/home/requirements.md]: (typo) communite

File: docs/home/requirements.md (hunk 2 above)
Comment (bmehendale) [reply docs/home/requirements.md]: Actually, in this case we don't want the resolved name. This is an example saying how the URL looks.

File: docs/home/requirements.md (hunk 3 above)
Comment (bmehendale) [reply docs/home/requirements.md]: good catch! will fix.

File: docs/home/README.md (hunk 1 above)
Comment (bmehendale) [reply docs/home/README.md]: good catch! will fix.

**Pull request merged**